src/View/Helper/TimeHelper.php

```diff
@@ -190,7 +215,7 @@ public function toUnix($dateString, $timezone = null)
      */
     public function toAtom($dateString, $timezone = null)
     {
-        $timezone = $timezone ?: date_default_timezone_get();
+        $timezone = $timezone ?: $this->defaultOutputTimezone;
```

Comment:
Atom strings should always end up in UTC. I don't think using the defaultOutputTimezone is correct here.